packages/server/src/config.ts

```diff
@@ -4,29 +4,44 @@ import { PERMISSIONS } from '../../lib/utils/constants';
 const config = {
   // URI for centrifuge node
   centrifugeUrl: env.CENTRIFUGE_URL || 'http://127.0.0.1:8082',
+  // The domain on which the application is hosted. Used for building links
+  // in emails
+  applicationHost: env.APPLICATION_HOST || 'http://gateway.centrifuge.io',
   // Port on which the application will run
   applicationPort: env.APPLICATION_PORT || '3001',
   sessionSecret: env.CENTRIFUGE_SESSION_SECRET || 'centrifuge',
+  email: {
+    user: env.CENTRIFUGE_EMAIL_CLIENT_USER || 'apikey',
+    password: env.CENTRIFUGE_EMAIL_SERVICE_APIKEY,
+    from: env.CENTRIFUGE_ADMIN_EMAIL || 'gateway@centrifuge.io',
+  },
   // We use replace to create a new database without changing the deployment config
   dbPath: env.DB_PATH ? env.DB_PATH.replace('db', 'db1') : './db',
   // Default admin user that will be created
   admin: {
     name: env.CENTRIFUGE_ADMIN_USER || 'admin',
-    email: env.CENTRIFUGE_ADMIN_EMAIL || 'test@test.org',
+    email: env.CENTRIFUGE_ADMIN_EMAIL || 'gateway@centrifuge.io',
```

Comment:
I decided to remove all the default values that can be secrets or different on from user to user. I think we should setup env vars in our .bash_profile. The ones I added: 
```
export CENTRIFUGE_URL
export APPLICATION_HOST
export CENTRIFUGE_EMAIL_SERVICE_APIKEY
export CENTRIFUGE_ADMIN_ACCOUNT=
export CENTRIFUGE_CHAIN_ID
export CENTRIFUGE_CHAIN_SECRET
export CENTRIFUGE_CHAIN_ADDRESS
export ETH_PROVIDER
export ETH_NETWORK
```